### ISSUE TITLE
Facilitate publishing a major version

### ISSUE THREAD

**Issue by peterdesmet** [gbif/ipt#1767]
When republishing a dataset, the following text is presented:

> You are about to publish a new minor version of this resource. If there have been scientifically significant changes to the resource since the last publication, you should cancel, assign this resource a new DOI, and then publish again.

I think this is somewhat unhelpful:

1. What is a scientifically significant change?
2. How do I assign a new DOI (if my resources are assigned a DOI via GBIF)?

Note: there is probably a larger discussion needed on DOIs and versioning.
- Should the data publisher decide if a version is minor or major or should that be automatically assigned by the IPT based on certain criteria (e.g. metadata update vs data update)?
- Should it be possible for a data publisher to edit/update metadata without creating a new (minor) version (cf. Zenodo)?
- Should there be a version-less DOI that always points to the most current one (cf. Zenodo)?

In the meantime, I think it would be good to update the help text to:

> You are about to publish a new minor version of this resource. If there have been scientifically significant changes to the resource since the last publication **and this IPT is configured to assign DOIs**, you should cancel, assign this resource a new DOI, and then publish again.

**Comment by peterdesmet** [comment]
Yeah, I find the current message confusing for an IPT that is not set up with DOIs, so I agree with:

> Or maybe the versions are irrelevant unless the IPT has assigned DOIs

For IPTs that do not assign DOIs, I would write:

> You are about to publish a new minor version of this resource.

**Comment by abubelinha** [comment]
>For IPTs that do not assign DOIs, I would write:
>>You are about to publish a new minor version of this resource.

Yes, I agree. 
All current DOIs wording is pretty unhelpful if the IPT does not assign DOIs.

**Comment by peterdesmet** [comment]
@mike-podolskiy90 note that the same phrase is used in the dialog box when republishing a Data Package. The DOI mention is confusing here too, and "You are about to publish a new public minor version of this resource." is incorrect, since Data Packages are published as major releases (1.0 -> 2.0 -> 3.0).

**Comment by mike-podolskiy90** [comment]
We simplified the message for Data package resources (there is no minor/major versions there, #2293 ), but it's still same confusing for DwC.

So we need to decide what to do.

**Comment by CecSve** [comment]
Relevant Discourse thread on the topic: https://discourse.gbif.org/t/when-to-assign-a-new-doi-to-an-existing-dataset/2319/3
